tests/internal/test_serverless.py

```diff
@@ -78,7 +80,40 @@ def test_not_azure_function_consumption_plan():
     assert in_azure_function() is False
 
 
-def test_slow_imports():
+standard_blocklist = [
+    "ddtrace.appsec._api_security.api_manager",
+    "ddtrace.appsec._iast._ast.ast_patching",
+    "ddtrace.internal.telemetry.telemetry_writer",
+    "email.mime.application",
+    "email.mime.multipart",
+    "logging.handlers",
+    "multiprocessing",
+    "importlib_metadata",
+    # These modules must not be imported because their source files are
+    # specifically removed from the serverless python layer.
+    # See https://github.com/DataDog/datadog-lambda-python/blob/main/Dockerfile
+    "ddtrace.appsec._iast._taint_tracking._native",
+    "ddtrace.appsec._iast._stacktrace",
+    "ddtrace.internal.datadog.profiling.libdd_wrapper",
+    "ddtrace.internal.datadog.profiling.ddup._ddup",
+    "ddtrace.internal.datadog.profiling.stack_v2._stack_v2",
+]
+expanded_blocklist = standard_blocklist + [
+    "importlib.metadata",
+]
+
+
+@pytest.mark.parametrize(
+    "package,blocklist",
+    [
+        ("ddtrace", expanded_blocklist),
+        ("ddtrace.contrib.internal.aws_lambda", expanded_blocklist),
+        ("ddtrace.contrib.internal.psycopg", expanded_blocklist),
+        # requests imports urlib3 which imports importlib.metadata
+        ("ddtrace.contrib.internal.requests", standard_blocklist),
```

Comment:
I would prefer if we just commented this line out with a comment than changing it. As it is now, it isn't actually testing what we need it to.

Reply:
Noted, I'll make that adjustment!

Reply:
It's commented out with a TODO note now.